src/transformers/file_utils.py

```diff
@@ -872,6 +874,9 @@ def hf_bucket_url(model_id: str, filename: str, revision: Optional[str] = None,
     its sha1 if stored in git, or its sha256 if stored in git-lfs. Files cached locally from transformers before v3.5.0
     are not shared with those new files, because the cached file's name contains a hash of the url (which changed).
     """
+    if subfolder is not None:
+        filename = f"{subfolder}/{filename}"
+
```

Comment:
looks like Python strings are value types so this should be safe @sgugger

Reply:
Yes, I have learned my lesson :-)

Reply:
(i wasn't sure either – always a good reflex to check I guess)